R/db_create_coverage_table.R

```diff
@@ -111,14 +133,33 @@ db_create_coverage_table <- function(ref_year_table,
     base::transform(pcn_region_code = "TOT") %>%
     data.table::as.data.table()
 
-  # Combine to a single table
-  out <- rbind(out_region, out_wld, out_tot)
+  # Income group coverage
+  out_inc <- dt %>%
+    dplyr::filter(incgroup_historical %in% c("Low income", "Lower middle income")) %>%
+    dplyr::group_by(reporting_year, incgroup_historical) %>%
+    dplyr::summarise(coverage = stats::weighted.mean(coverage, pop)) %>%
+    dplyr::group_by(reporting_year) %>%
+    dplyr::summarise(coverage = mean(coverage)) %>%
+    dplyr::mutate(incgroup_historical = "LIC/LMIC") %>%
```

Comment:
What is this for? Just worried about hard coded values...

Reply:
I don't think there is any hardcoded values in these four lines of code? The reason we are calculating coverage by historical income groups is because the censoring of WLD year depends on coverage in LIC/LMIC countries, in addition global coverage (this is a "new" rule that was included in PCN in 2021).